src/main/java/seedu/address/logic/commands/FavListCommand.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Lists all favourite persons in the address book to the user.
+ */
+public class FavListCommand extends Command {
+    public static final String MESSAGE_SUCCESS = "You have 0 favourite profile in your list.\n";
```

Comment:
Would it be better (and less hardcode-worthy) to use a String format here instead?